src/components/content/credential/Credential.tsx

```diff
@@ -0,0 +1,228 @@
+        csp: '' as CredentialVariables.csp,
+        description: '',
+        name: '',
+        timeToLive: 0,
+        type: '' as CredentialVariables.type,
+        variables: [],
+        xpanseUser: '',
+    });
+
+    const columns: ColumnsType<CredentialVariables> = [
+        {
+            title: 'Csp',
+            dataIndex: 'csp',
+        },
+        {
+            title: 'User',
+            dataIndex: 'xpanseUser',
+        },
+        {
+            title: 'Name',
+            dataIndex: 'name',
+        },
+        {
+            title: 'Type',
+            dataIndex: 'type',
+        },
+        {
+            title: 'Description',
+            dataIndex: 'description',
+        },
+        {
+            title: 'Operation',
+            dataIndex: 'operation',
+            render: (text: string, record: CredentialVariables) => {
+                return (
+                    <>
+                        <Space size='middle'>
+                            <Button type='primary' icon={<FullscreenOutlined />} onClick={() => details(record)}>
+                                Details
+                            </Button>
+                            <Popconfirm
+                                title='Delete the Credential'
+                                description='Are you sure to delete the Credential?'
+                                okText='Yes'
+                                cancelText='No'
+                                onConfirm={() => deleteCredential(record)}
+                            >
+                                <Button type='primary' icon={<MinusCircleOutlined />}>
+                                    delete
+                                </Button>
+                            </Popconfirm>
+                            <Button
+                                type='primary'
+                                icon={<InfoCircleOutlined />}
+                                onClick={() => updateCredential(record)}
+                            >
+                                update
+                            </Button>
+                        </Space>
+                    </>
+                );
+            },
+        },
+    ];
+
+    const addCredential = () => {
+        setIsAddOpen(true);
+    };
+
+    const deleteCredential = (record: CredentialVariables) => {
+        void CredentialsManagementService.deleteCredential(record.csp, record.xpanseUser, record.type)
+            .then(() => {
+                getTipInfo('success', 'delete success!');
+                getCredentials();
+            })
+            .catch(() => {
+                getTipInfo('error', 'delete failed!');
+            });
+    };
+
+    const updateCredential = (record: CredentialVariables) => {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        const createCredential: CreateCredential = JSON.parse(JSON.stringify(record));
+        createCredential.timeToLive = 3600;
```

Comment:
this is the default value already in the backend. 
In the UI, we must ask user to enter this value  as optional field. Only if  user enters any value, we must send it  in the API request.

Reply:
i still dont see this

Reply:
i still dont see an option for user to enter time to live.